### DISCUSSION TITLE
What do you want to learn next?

### DISCUSSION THREAD

**Discussion by marcduiker** [marcduiker/azure-functions-university#218] [General]
What would you like to learn next about Azure Functions?

**Comment by DamolAAkinleye** [comment]
Can you do a video on writing a performant Azure Durable Function using Python and Asyncio libraries

**Reply by marcduiker** [reply]
Good suggestion! I'll ask around who can help me with Durable Functions lessons for Python, similar to the TypeScript lessons for Durable Functions. Are you looking for some advice of using asyncio in activity functions?
